shielddash/core/static/js/app/api/study-api.js

```diff
@@ -5,18 +5,17 @@ import {
   getStudyFetching, getStudySuccess, getStudyFailure
 } from '../actions/study-actions';
 
-const mockEndpoints = {
-  GET_STUDIES: 'http://localhost:3009/studies',
-  GET_STUDY: 'http://localhost:3009/studys/'
+const devEndpoints = {
+  GET_STUDIES: 'http://localhost:8000/studies/',
+  GET_STUDY: 'http://localhost:8000/studies/'
 };
 
 const prodEndpoints = {
   GET_STUDIES: 'https://shielddash.herokuapp.com/studies/',
   GET_STUDY: 'https://shielddash.herokuapp.com/studies/'
 }
 
-// TODO: Check node environment to set this.
-const endpoints = process.env.NODE_ENV === 'dev' ? mockEndpoints : prodEndpoints;
+const endpoints = __PROD_API__ ? prodEndpoints : devEndpoints;
```

Comment:
Sure, the reason is that we don't have `process.env` available when the JavaScript file is evaluated in the browser which made the current code no-op.

`process` is a Node module and `process.env` a mapping to the environment variables that Node processes are started with. But once we've built the bundle file we won't have access to this anymore and in my tests the gulp bundling tasks file didn't replace `process.env.NODE_ENV === 'dev'` with the evaluated value as I anticipated. I'm not sure if that was the original intention, or if it's a transfer issue from the separate frontend repo, but I'd be happy to change it to whatever works best for you. Do you agree with the default to being 

The webpack approach is basically to manually define a global value that webpack will look for and replace with during the build step. I just called it `__PROD_API__` to make sure we know not to touch it, but I guess we can call it whatever we like.